### PR TITLE
OCPBUGS-13855:Agent TUI release note

### PR DESCRIPTION
**Pull request by skopacz1**
[OCPBUGS-13855](https://issues.redhat.com/browse/OCPBUGS-13855)

Version: 4.13 only

This PR adds a release note entry in the New features and enhancement section describing the agent-tui/agent console application that is now a part of Agent-based installations.

QE review:
- [x] QE has approved this change.

Preview: [Release Notes](http://file.bos.redhat.com/skopacz/OCPBUGS-13855/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-agent-console-application)

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc
Comment (andfasano) [inline release_notes/ocp-4-13-release-notes.adoc:286]:
```suggestion
For installations of {product-title} {product-version} using the Agent-based Installer, a console application (with a textual user interface) performs a pull check early in the installation process to verify that the current host can retrieve the configured release image. The console application supports the user in troubleshooting the problem in case of issue, and it also allows to modify directly the network configuration in case of need.
```

Comment (andfasano) [reply release_notes/ocp-4-13-release-notes.adoc:286]: Just my suggestions, feel free to rework them as per your convenience

**Comment by mhanss** [comment]
lgtm

**Review by bergerhoffer — COMMENTED** [review]
This update LGTM, but just wanted to see if you checked with @kalexand-rh on whether this needs to go through change management approval first (since it's updating an existing set of release notes)?

**Comment by kalexand-rh** [comment]
Since this is an accidental omission less than a week from GA and it's for feature work instead of bringing up a known issue, a literal interpretation of the change management guidelines says "no" to CM.

**Comment by skopacz1** [comment]
Thanks @bergerhoffer and @kalexand-rh for the question and discussion about change management!

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**
